plumbum/cli/termsize.py

```diff
@@ -71,18 +72,19 @@ def _get_terminal_size_tput():  # pragma: no cover
         return None
 
 
-def _ioctl_GWINSZ(fd):
+def _ioctl_GWINSZ(fd: int) -> Optional[Tuple[int, int]]:
     yx = Struct("hh")
     try:
         import fcntl
         import termios
 
-        return yx.unpack(fcntl.ioctl(fd, termios.TIOCGWINSZ, "1234"))
+        # TODO: Clean this up. Problems could be hidden by the broad except.
+        return yx.unpack(fcntl.ioctl(fd, termios.TIOCGWINSZ, b"1234"))  # type: ignore[return-value]
```

Comment:
This was the possible bug, it accepted a byte string in Python 3, not a unicode string.